### ISSUE TITLE
Mention autograd.pause() in autograd section

### ISSUE THREAD

**Issue by linmx0130** [zackchase/mxnet-the-straight-dope#297]
Inplace operations (+=, -=, x[:]=, etc) are not supported when recording with autograd. But in object detection, we need to use inplace operations to calcuate regression target, which does not need gradient in fact. There are also numerous situations that we do not need gradient calculated by gluon. 

I think mentioning autograd.pause() in Chapter 1 is important since it is a crucial and useful part of autograd.

**Comment by roachsinai** [comment]
Yes, maybe I understand the function of pause by this issue. And I'm learning your project `Yet Another MXnet DETection `.

谢谢
